app/controllers/claims_controller.rb

```diff
@@ -30,7 +30,7 @@ def show
   def update
     current_claim.attributes = claim_params
     current_claim.reset_dependent_answers
-    current_claim.eligibility.reset_dependent_answers
+    current_claim.eligibility.reset_dependent_answers unless current_claim.policy == EarlyCareerPayments
```

Comment:
Why are we doing this?

Reply:
@nomasprime This was what I talked about in the walkthrough yesterday, I could have opted to put the method into the `EarlyCareerPayments::Eligibility` class. I opted for this instead

Reply:
Understand this now.

Personally would have opted for the latter because conditional logic to decide whether to call something is usually a smell. Can't remember technical term off the top of my head but see null object for something similar.